overrides/main.html

```diff
@@ -25,3 +25,49 @@
 {% endblock %}
 
 
+
+{% block analytics %}
+<!-- RightMessage -->
+<script type="text/javascript"> 
+  (function(p, a, n, d, o, b, c) {
+      o = n.createElement('script'); o.type = 'text/javascript'; o.async = true; o.src = 'https://tb.rightmessage.com/'+p+'.js';
+      b = n.getElementsByTagName('script')[0]; d = function(h, u, i) { var c = n.createElement('style'); c.id = 'rmcloak'+i;
+          c.type = 'text/css'; c.appendChild(n.createTextNode('.rmcloak'+h+'{visibility:hidden}.rmcloak'+u+'{display:none}'));
+          b.parentNode.insertBefore(c, b); return c; }; c = d('', '-hidden', ''); d('-stay-invisible', '-stay-hidden', '-stay');
+      setTimeout(o.onerror = function() { c.parentNode && c.parentNode.removeChild(c); }, a); b.parentNode.insertBefore(o, b);
+  })('767950886', 20000, document);
+  </script>
+  {% set analytics = config.google_analytics %}
+  <link rel="preconnect dns-prefetch" href="https://www.google-analytics.com">
+  
+  <script type="application/javascript">
+    window.ga = window.ga || function() {
+      (ga.q = ga.q || []).push(arguments)
+    }
+    ga.l = +new Date
+  
+    /* Set up integration and send page view */
+    ga("create", "{{ analytics[0] }}", "{{ analytics[1] }}")
+    ga("set", "anonymizeIp", true)
+    ga("send", "pageview")
+  
+    /* Register handler to log search on blur */
+    document.addEventListener("DOMContentLoaded", function() {
+      if (document.forms.search) {
+        var query = document.forms.search.query
+        query.addEventListener("blur", function() {
+          if (this.value) {
+            var path = document.location.pathname;
+            ga("send", "pageview", path + "?q=" + this.value)
+          }
+        })
+      }
+    })
+  
+    /* Send page view on document switch (instant loading, custom event) */
+    document.addEventListener("DOMContentSwitch", function() {
+      ga("send", "pageview")
+    })
+  </script>
+  <script async src="https://www.google-analytics.com/analytics.js"></script>
```

Comment:
So this is really necessary?
Not like I block it at least 3 times.

Reply:
I'm looking into alternatives, I agree Google is "icky". Matomo looks good, but their pricing is confusing. I've also experimented with Netlify Analytics, but I found that it didn't provide enough detail.. I'm open to suggestions ;)

Reply:
I just selfhost matomo.

Reply:
I considered that, but the [list of features](https://matomo.org/pricing/) you _don't_ get on self-hosted instances seemed ... overwhelming. Have you felt limited by the self-hosted version?

Reply:
To be honest I never really digged into the stats to deep, so I didn't notice it.